.vscode/settings.json

```diff
@@ -3,110 +3,37 @@
   },
-  "[json][jsonc][markdown][yaml]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
-  },
   "[javascript][typescript]": {
-    "editor.formatOnSave": false,
-    "editor.defaultFormatter": "esbenp.prettier-vscode",
     "editor.codeActionsOnSave": {
-      "source.unusedImports": "explicit",
-      "source.fixAll": "explicit",
-      "source.formatDocument": "always"
-    }
-  },
-  "editor.formatOnSave": true,
-  "eslint.enable": true,
-  "eslint.codeAction.showDocumentation": {
-    "enable": true
-  },
-  "eslint.codeActionsOnSave.mode": "all",
-  "eslint.lintTask.enable": true,
-  "eslint.onIgnoredFiles": "warn",
-  "eslint.workingDirectories": [
-    {
-      "mode": "auto"
-    }
-  ],
-  "eslint.validate": ["javascript", "typescript", "json", "jsonc"],
-  "files.exclude": {
-    "**/.DS_Store": true,
-    "**/.git": true,
-    "**/node_modules": true,
-    "**/dist": true,
-    "tracerbench-results": true
-  },
-  "files.insertFinalNewline": true,
-  "files.trimTrailingWhitespace": true,
-  "files.watcherExclude": {
-    "**/.git/objects/**": true,
-    "**/.git/subtree-cache/**": true
-  },
-  "javascript.updateImportsOnFileMove.enabled": "always",
-  "typescript.updateImportsOnFileMove.enabled": "always",
-  "javascript.preferences.importModuleSpecifier": "project-relative",
-  "typescript.preferences.importModuleSpecifier": "project-relative",
-  "typescript.preferences.importModuleSpecifierEnding": "auto",
-  "typescript.preferences.useAliasesForRenames": false,
-  "typescript.tsdk": "node_modules/typescript/lib",
-  "typescript.tsserver.experimental.enableProjectDiagnostics": true,
-  "typescript.workspaceSymbols.scope": "currentProject",
-  "typescript.experimental.updateImportsOnPaste": true,
-  "eslint.problems.shortenToSingleLine": true,
-  "typescript.experimental.expandableHover": true,
-  "inline-bookmarks.expert.custom.words.mapping": {
-    "warn": ["@premerge(\\s|$)"],
-    "active": ["@active(\\s|$)"],
-    "fixme": ["@fixme(\\s|$)"]
-  },
-  "inline-bookmarks.expert.custom.styles": {
-    "active": {
-      // teal
-      "gutterIconColor": "#009999",
-      "overviewRulerColor": "rgba(0, 150, 150, 0.7)",
-      "light": {
-        "fontWeight": "bold",
-        "color": "#009999"
-      },
-      "dark": {
-        "fontWeight": "bold",
-        "color": "#00ffff"
-      }
+      "source.fixAll": "always",
+      "source.fixAll.eslint": "always",
+      "source.fixAll.ts": "always",
+      "source.formatDocument": "always",
     },
-    "fixme": {
-      // yellow
-      "gutterIconColor": "#990000",
-      "overviewRulerColor": "rgba(150, 0, 0, 0.7)",
-      "light": {
-        "fontWeight": "bold",
-        "color": "#990000"
-      },
-      "dark": {
-        "fontWeight": "bold",
-        "color": "#ff0000"
-      }
+    "editor.rulers": [
+      100
+    ],
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "editor.formatOnSave": false
+  },
+  "[json][jsonc][markdown][yaml]": {
```

Comment:
Same deal with JSON. Note that JSON and JSONC are linted via `eslint` (via `eslint-plugin-jsonc`) in
this repository, so `fixAll` is meaningful.